### ISSUE TITLE
Imagick PDF to JPG conversion is broken

### ISSUE THREAD

**Issue by Ruesa18** [whatwedo/docker-base-images#65]
In Version 2.8 (and probably onwards?) the imagick integration (or libimagemagick rather) doesn't work as it did in older versions.
If you try to convert a PDF into a JPG image, PHP will throw an ImagickException like this:
`no decode delegate for this image format 'PDF' @ error/constitute.c/ReadImage/746`

Code used to provoke this error:
```php
$pdfThumb = new \imagick();
$pdfThumb->setResolution(800, 800);
$pdfThumb->readImageFile(fopen('test.pdf'));
$pdfThumb->setImageCompression(\imagick::COMPRESSION_JPEG);
$pdfThumb->setImageCompressionQuality(40);
$pdfThumb->setImageFormat('jpg');
$pdfThumb->writeImage('example.jpg');
```

**Comment by xarem** [comment]
@Ruesa18 fixed in v2.8 and v2.9, please pull the image again and test.

**Comment by Ruesa18** [comment]
@xarem Works as far as I can tell 👍 
Thanks for the fix 🙏
